src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -963,8 +963,8 @@ const static std::unordered_map<std::string, uint32_t> ActionNameToType = {
     { "footpathplace", GAME_COMMAND_PLACE_PATH },
     { "footpathplacefromtrack", GAME_COMMAND_PLACE_PATH_FROM_TRACK },
     { "footpathremove", GAME_COMMAND_REMOVE_PATH },
-    { "footpathsceneryplace", GAME_COMMAND_PLACE_FOOTPATH_SCENERY },
-    { "footpathsceneryremove", GAME_COMMAND_REMOVE_FOOTPATH_SCENERY },
+    { "footpathadditionplace", GAME_COMMAND_PLACE_FOOTPATH_SCENERY },
```

Comment:
Could you also rename the game command enum members accordingly?

Reply:
Sure, done.